osu.Game.Rulesets.Osu/Difficulty/Evaluators/AimEvaluator.cs

```diff
@@ -88,9 +88,12 @@ public static double EvaluateDifficultyOf(DifficultyHitObject current, bool with
                     }
 
                     // Penalize wide angles if they're repeated, reducing the penalty as the lastAngle gets more acute.
```

Comment:
The comment would need to be adjusted for the changes made to the line below (`lastLastAngle` instead of `lastAngle`) I think?

Reply:
I agree. I'll get this fixed in next commit.